build.sbt

```diff
@@ -468,6 +468,7 @@ lazy val kafka = (project in engine("kafka")).
     name := "nussknacker-kafka",
     libraryDependencies ++= {
       Seq(
+        "javax.validation" % "validation-api" % javaxValidationApiV,
```

Comment:
let's remove it for next PR :)

Reply:
I've added it in another PR: https://github.com/TouK/nussknacker/pull/809. There I will remove it.